db/migrate/20200923101308_create_vendor_api_traces.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVendorAPITraces < ActiveRecord::Migration[6.0]
+  def change
+    create_table :vendor_api_traces do |t|
+      t.jsonb     :headers
+      t.jsonb     :request
+      t.jsonb     :response
+      t.integer   :status
+      t.bigint    :request_time
+
```

Comment:
We also want a way to match the requests to providers, so we'll need a reference to the token that made the request I think.

Reply:
That leads to the question what to do with requests with a wrong token. Should we ignore those?

Reply:
The app will serve an error response in the usual way, which would be good to know about (maybe someone's integration is broken)

Reply:
But if the token is wrong, we won't know who made the request I think.

Reply:
We might have the headers including forwarded-for and possibly user-agent?

Reply:
But maybe this is just one for Kibana? Could get noisy too

Reply:
Should we persist a column with values something like...
```ruby
provider_uuid: Digest::MD5.hexdigest(request_headers.delete('Authorization'))`
```